### PR TITLE
fix conditional DETR AP and Log

### PR DESCRIPTION
**Pull request by LYMDLUT**
I'm working together with @jshilong, @Li-Qingyun, and @KeiChiTse to refactor the DETR-like algorithms in MMDetection 3.x, using a more lightweight and easy-to-read code style.

This PR is to fix the AP and Log result in conditional DETR, I update the log from 150 epoch to 50 epoch. And the AP update from 40.9 in original repo to 41.1

### CODE REVIEW

**Comment by LYMDLUT** [comment]
https://github.com/LYMDLUT/mmdetection/releases/download/untagged-b75ab9b3b045bbfb5c35/conditional-detr_r50_8xb2-50e_coco_20221121_180202.log.json

This is the 50e log of conditional DETR，please help me to update it to our server

**Comment by jshilong** [comment]
> I'm working together with @jshilong, @Li-Qingyun, and @KeiChiTse to refactor the DETR-like algorithms in MMDetection 3.x, using a more lightweight and easy-to-read code style.
> 
> This PR is to fix the AP and Log result in conditional DETR, I update the log from 150 epoch to 50 epoch. And the AP update from 40.9 in original repo to 41.1

@hhaAndroid Could you help to update this log file?

**Comment by hhaAndroid** [comment]
@LYMDLUT I have uploaded

File: configs/conditional_detr/README.md [hunk 1]
```diff
@@ -23,9 +23,9 @@ Our conditional DETR learns a conditional spatial query from the decoder embeddi
...
+| Backbone |      Model       | Lr schd | Mem (GB) | Inf time (fps) | box AP |                      Config                       |                                                                                         Download                                                                                          |
+| :------: | :--------------: | :-----: | :------: | :------------: | :----: | :-----------------------------------------------: | :---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------: |
+|   R-50   | Conditional DETR |   50e   |          |                |  41.1  | [config](./conditional-detr_r50_8xb2-50e_coco.py) | [model](https://download.openmmlab.com/mmdetection/v3.0/conditional_detr/conditional-detr_r50_8xb2-50e_coco/conditional-detr_r50_8xb2-50e_coco_20221121_180202-c83a1dc0.pth) \| [log](<>) |
```
Comment (hhaAndroid) [inline configs/conditional_detr/README.md:28]: log link

Comment (LYMDLUT) [reply configs/conditional_detr/README.md:28]: https://github.com/LYMDLUT/mmdetection/releases/download/untagged-b75ab9b3b045bbfb5c35/conditional-detr_r50_8xb2-50e_coco_20221121_180202.log.json

Comment (LYMDLUT) [reply configs/conditional_detr/README.md:28]: Thanks a lot

Comment (hhaAndroid) [reply configs/conditional_detr/README.md:28]: @LYMDLUT Please update the PR

Comment (LYMDLUT) [reply configs/conditional_detr/README.md:28]: > @LYMDLUT Please update the PR

please provide the log link

Comment (hhaAndroid) [reply configs/conditional_detr/README.md:28]: https://download.openmmlab.com/mmdetection/v3.0/conditional_detr/conditional-detr_r50_8xb2-50e_coco/conditional-detr_r50_8xb2-50e_coco_20221121_180202.log.json

**Review by hhaAndroid — APPROVED** [review]

**Pull request merged**